shell/utils/platform.js

```diff
@@ -46,6 +46,7 @@ export const isBlink = userAgent.includes('Chrome/');
 export const isWebKit = !isBlink && userAgent.includes('AppleWebKit/');
 export const isSafari = !isBlink && userAgent.includes('Safari/');
 export const isMobile = /Android|webOS|iPhone|iPad|iPod|IEMobile/i.test(userAgent);
+export const isChromium = !!window.chrome;
```

Comment:
this doesn't appear to be used anywhere...

Reply:
I forgot do remove it, thanks.